HW_6/HW_5a.py

```diff
@@ -0,0 +1,144 @@
+
+import datetime
+import random
+import datefinder
+import HW_3_updated as normalization
+import sys, os
+
+sys.path.append('C:\\Users\\Kateryna_Kulmatytska\\PycharmProjects\\Python_HW\\HW_6')
```

Comment:
Your code shouldn't contain any user-based paths. 
The standard module **os.path** contains all functions in order to parse.
More about that: https://pythonworld.ru/moduli/modul-os-path.html